src/test/resources/sql/location-init.sql

```diff
@@ -1,9 +1,13 @@
+INSERT INTO  country
+VALUES ('UA', 'Ukraine');
```

Comment:
country lookup table should not be modified in tests

Reply:
Removed